core/frontend/src/IModelApp.ts

```diff
@@ -119,6 +121,8 @@ export interface IModelAppOptions {
   /** If present, supplies the [[UiAdmin]] for this session. */
   uiAdmin?: UiAdmin;
   rpcInterfaces?: RpcInterfaceDefinition[];
+  /** @internal // SWB What should this be? */

```

Comment:
Lets mark this as `beta` to start with since we'll want everyone to set it once they move to 3.0.

Before we move it to `public` we should have some sort of testing around it...

Reply:
What do you think of renaming this to `realityData` or `realityDataAccess` to follow the precedent set by the `HubAccess`?

Reply:
I think `realityDataAccess` would fit that precedent well.